pkg/kor/all.go

```diff
@@ -23,6 +23,11 @@ type ResourceDiff struct {
 	diff         []string
 }
 
+type ResourceDiff2 struct {
+	resourceType string
+	diff         interface{}
```

Comment:
I think that instead of ising an interface why not always use the `ResourceInfo` and depending on the flag, set/display the reson or not...

Reply:
Hmm. This might be significantly cleaner. I'll give it a try